app/templates/components/ui-table/cell/events/view/tickets/attendees/cell-price.hbs

```diff
@@ -1,3 +1,3 @@
 <span>
-  {{currency-symbol paymentCurrency}} {{number-format record.ticket.price}}
+  {{currency-symbol paymentCurrency}} {{number-format record.order.amount}}
```

Comment:
@Anupam-dagar @CosmicCoder96 I think it should be order amount by number of tickets bought.

Reply:
@harsh-jindal I believe in `order.amount` the final discounted price is stored, so @Anupam-dagar is correct in the implementation

Reply:
@harsh-jindal it works for multiple tickets too.